testing/ltp/Makefile

```diff
@@ -103,9 +119,23 @@ CFLAGS       += -I$(LTP_UNPACK)
 CFLAGS       += -I$(TESTDIR)/include
 endif
 
+# Relax warning checks to avoid expected compile errors:
+CFLAGS += -Wno-strict-prototypes -Wno-return-type -Wno-format -Wno-uninitialized
+CFLAGS += -Wno-unused-variable -Wno-unused-function -Wno-unused-but-set-variable -Wno-unused-value
+
+# Should be removed if possible in the future
+CFLAGS += -Wno-incompatible-pointer-types -Wno-overflow -Wno-int-to-pointer-cast
+
+# This should be defined in unistd.h
+CFLAGS += -D_POSIX_TIMEOUTS=1
```

Comment:
it's better to add this macro to nuttx repo

Reply:
https://github.com/apache/incubator-nuttx/pull/3515

Reply:
This patch is merged, _POSIX_TIMEOUTS can be removed from here.

Reply:
Updated.